src/styles/blocks/is-active.scss

```diff
@@ -0,0 +1,13 @@
+.is-active {
```

Comment:
this style belongs to nav so it would be better to place it there

Reply:
the test says that .is-active must be a separate class, so it's impossible to implement that way